### PR TITLE
Add konf

### LINKED ISSUE #18: Add konf configuration to make this live
Let's add the konf configuration with all the relevant secrets to make this live!

### PR DESCRIPTION
**Pull request by andesol**
## Done 

- Add `site.yml` file

## Issue

Fixes #18

### CODE REVIEW

File: konf/site.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+domain: specs.canonical.com
+
+image: prod-comms.docker-registry.canonical.com/specs.canonical.com
+
+env:
+  - name: SENTRY_DSN
+    value: https://f51f18282e7d44bebe6cc268ba96f7a2@sentry.is.canonical.com//49
+
+  - name: GOOGLE_PRIVATE_KEY_ID
```
Comment (goulinkh) [inline konf/site.yml]: This has to be the same name as [here](https://github.com/canonical-web-and-design/specs.canonical.com/blob/main/webapp/spreasheet.py#L10):
```py
os.getenv("PRIVATE_KEY_ID")
```

File: konf/site.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    secretKeyRef:
+      key: private-key-id
+      name: google-private-key-id
+
+  - name: GOOGLE_PRIVATE_KEY
```
Comment (goulinkh) [inline konf/site.yml]: Same here

**Review by goulinkh — CHANGES_REQUESTED** [review]

**Comment by andesol** [comment]
Ops, guess I forgot my own naming conventions...

File: konf/site.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+domain: specs.canonical.com
+
+image: prod-comms.docker-registry.canonical.com/specs.canonical.com
+
+env:
+  - name: SENTRY_DSN
+    value: https://f51f18282e7d44bebe6cc268ba96f7a2@sentry.is.canonical.com//49
```
Comment (tbille) [inline konf/site.yml]:
```suggestion
    value: https://e38013698989400884cce7bd9d862af3@sentry.is.canonical.com//51
```

I just created a project for [specs.canonical.com on sentry](https://sentry.is.canonical.com/canonical/specs-canonical-com/). You can use this variable instead :)

**Review by tbille — APPROVED** [review]
One small change and feel free to merge.

**Pull request merged**